utilities/rocAL/rocAL_video_unittests/README.md

```diff
@@ -0,0 +1,132 @@
+
+The sample video files and folder are available in the In the following path : [video and sequence samples](https://github.com/GPUOpen-ProfessionalCompute-Libraries/MIVisionX-data/tree/main/rocal_data/video_and_sequence_samples).
+
+The data samples can be downloaded from the MIVisionX-data repository.
+
+```
+git clone https://github.com/GPUOpen-ProfessionalCompute-Libraries/MIVisionX-data.git
+```
+
+## Description
+
+### Test Cases
+
+The following test cases are supported in this unittest:
+1. Video Reader - Reads the video file/folder and returns a sequence of frames.
+2. Video Reader Resize (reader followed by resize augmentation)  - Reads the video file/folder and returns a sequence of re-sized frames.
+3. Sequence Reader - Reads the folder of images and returns a sequence of images.
+
+### Arguments used in test script
+
+INPUT_PATH : Input passed by the user. It can be a video file path, folder path containing videos or a text file.
+
+  NOTE:
+
+  * Inputs for cases 1 and 2 - Video file / folder containing videos
+  * Input for case 3 - Folder containing sequence of images [sample folder](https://github.com/GPUOpen-ProfessionalCompute-Libraries/MIVisionX-data/tree/main/rocal_data/video_and_sequence_samples/sequence)
+
+READER_CASE : Value passed can be 1/2/3 depending upon the selected reader (default value : 1).
+
+SAVE_FRAMES : Saves the output frames or avi files in the build/output_frames folder.
+
+DEVICE : CPU:0/GPU:1 device is supported.
+
+HARDWARE_DECODE_MODE : Uses Hardware decoder if set to true. 
+
+SHUFFLE : Shuffles the sequences if set to true.
+
+BATCH SIZE : Number of sequences in a batch.
+
+SEQUENCE LENGTH : Number of frames in a sequence.
+
+STEP : The frame interval between sequences.
+
+STRIDE : The frame interval between frames within a sequence.
+
+RESIZE_WIDTH : Resize width value.
+
+RESIZE_HEIGHT : Resize height value.
+
+FILELIST_FRAMENUM : If set to true the inputs from text file will be considered as frame numbers otherwise they will be considered as timestamps.
+
+ENABLE_METADATA : If set to true, prints the labels and names of the associated frames in the sequence.
+
+ENABLE_FRAME_NUMBER : If set to true, prints the starting frame number of the sequences.
+
+ENABLE_TIMESTAMPS : If set to true, prints the timestamp of each frame in the sequences.
+
+ENABLE_SEQUENCE_REARRANGE : If set to true, the frames in each sequence will be rearranged in the order specified by the user. The order should contain values in the range of [0, sequence_length)
+
+## Test case examples
+
+**Example 1: Video Reader**
+
+> ./testScript.sh <[path/to/test_frame_num.mp4](https://github.com/GPUOpen-ProfessionalCompute-Libraries/MIVisionX-data/blob/main/rocal_data/video_and_sequence_samples/test_frame/test_frame_num.mp4)> 1
+
+Arguments to be modified in testScript.sh to get the following output:
+
+- BATCH_SIZE=2
+- SEQUENCE_LENGTH= 3
+- STEP=3
+- STRIDE=5
+
+![video_reader.png](./samples/video_reader.png)
```

Comment:
@fiona-gladwin can you check this output image? I have
0 5 10
3 8 13
Which makes sense. not sure how you have 14

Reply:
@kiritigowda @rrawther I don't see any hang issues. Just need to cross check output from my previous comment before merging

Reply:
@LakshmiKumar23 I have replaced the image with the correct sequence please check